docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -1,7 +1,7 @@
 # wstETH rollup bridging guide
 
 :::warning Disclaimer
-This guide provides recommendations provided by the NEW workgroup. NEW is not a representative of the Lido DAO and by giving the feedback NEW makes no warranties, express or implied, and disclaims all implied warranties, including any warranty of the likelihood of the recognition or rejection by the Lido DAO and representation.
+This guide provides recommendations provided by the [Network Expansion Committee (NEC)](https://snapshot.org/#/lido-snapshot.eth/proposal/0x7cdf1af7cfeb472ae202c45fb6d7e952bb34bfcbc82113549986b2bc2d5f54c5). NEC is not a representative of the Lido DAO and by giving the feedback NEC makes no warranties, express or implied, and disclaims all implied warranties, including any warranty of the likelihood of the recognition or rejection by the Lido DAO and representation.
```

Comment:
NEC is a representative committee of the DAO, though with a limited scope :thinking: 

> This proposal aims to establish a Network Expansion Committee that will be assigned to formally recognize (w)stETH token bridging endpoints and denominations on new networks as canonical, acting on behalf of Lido DAO.